luigi/parameter.py

```diff
@@ -394,11 +396,23 @@ class DateMinuteParameter(DateHourParameter):
     Parameter whose value is a :py:class:`~datetime.datetime` specified to the minute.
 
     A DateMinuteParameter is a `ISO 8601 <http://en.wikipedia.org/wiki/ISO_8601>`_ formatted
-    date and time specified to the minute. For example, ``2013-07-10T19H07`` specifies July 10, 2013 at
+    date and time specified to the minute. For example, ``2013-07-10T1907`` specifies July 10, 2013 at
     19:07.
     """
 
-    date_format = '%Y-%m-%dT%HH%M'  # ISO 8601 is to use 'T' and 'H'
+    date_format = '%Y-%m-%dT%H%M'
+    deprecated_date_format = '%Y-%m-%dT%HH%M'
+
+    def parse(self, s):
+        try:
+            return super(DateMinuteParameter, self).parse(s)
+        except ValueError:
+            warnings.warn(
+                'Using "H" between hours and minutes is deprecated, omit it instead.',
+                DeprecationWarning,
+                stacklevel=2
+            )
+            return datetime.datetime.strptime(s, self.deprecated_date_format)
```

Comment:
this will throw a deprecation warning for any misspecified string – maybe makes more sense to try deprecated first